app/build.gradle

```diff
@@ -15,14 +15,15 @@ jar {
 dependencies {
     implementation project(':api')
     compileOnly 'run.halo.app:api'
-    implementation 'org.dom4j:dom4j:2.1.3'
+    implementation 'org.dom4j:dom4j:2.1.4'
```

Comment:
The dom4j must be upgrade or log4j configuration cannot be resolved while running tests.